### DISCUSSION TITLE
线刷包发布帖 - 最新V1.0.0.94

### DISCUSSION THREAD

**Discussion by mi-CyberDog** [MiRoboticsLab/cyberdog_ros2#133] [Announcements]
### 使用方法

线刷包刷机方法可参考[Wiki](https://github.com/MiRoboticsLab/cyberdog_ros2/wiki/%E5%A6%82%E4%BD%95%E7%BA%BF%E5%88%B7%E9%93%81%E8%9B%8B)

### 下载地址

- [V1.0.0.66](https://cdn.cnbj2m.fds.api.mi-img.com/cyberdog-package/build/athena_foxy_2021.08.24_emmc_nvme_V1.0.0.66.20210824_release_bbcc37a86a.tgz): 首次发布版本
- [V1.0.0.82](https://cdn.cnbj2m.fds.api.mi-img.com/cyberdog-package/build/athena_foxy_2021.09.26_emmc_nvme_V1.0.0.82_release_45bed14190.tgz)
- [V1.0.0.94](http://cdn.cnbj2m.fds.api.mi-img.com/cyberdog-package/build/athena_foxy_2022.01.14_emmc_nvme_V1.0.0.94_release_b1b4a851ca.tgz)

**Comment by yangyh75** [comment]
一直找不到线刷94包的下载，电量为0，不能OTA升级，前天通过控制台发假电量成功下载OTA包，APP不能升级，最后是通过控制台完成OTA升级的。

**Reply by hhbbww83** [reply]
老哥你好，我也遇到了和你一样的问题，也是电量为0无法升级到v1.0.0.94。能否请教你一下具体做法？以及请问升级后电量显示会正常吗

**Comment by tarikbeijing** [comment]
您好，您能告诉我在哪里可以找到与V1.0.0.94版本相关的开源软件吗？在github上，它不是,https://partner-gitlab.mioffice.cn/cyberdog它也不是V1.0.0.94版本。感谢您的帮助。

**Comment by yangyh75** [comment]
你好：

    

最简单的方法是直接下载094的包刷机，Linux下刷机，这个需要根据官方文档（https://github.com/MiRoboticsLab/cyberdog_ros2/wiki/%E5%A6%82%E4%BD%95%E7%BA%BF%E5%88%B7%E9%93%81%E8%9B%8B）来就好了。


094刷机包：wget https://cdn.cnbj2m.fds.api.mi-img.com/cyberdog-package/build/athena_foxy_2022.01.14_emmc_nvme_V1.0.0.94_release_b1b4a851ca.tgz


***@***.***
 
发件人： hhbbww83
发送时间： 2024-09-26 17:05
收件人： MiRoboticsLab/cyberdog_ros2
抄送： yangyh75; Comment
主题： Re: [MiRoboticsLab/cyberdog_ros2] 线刷包发布帖 - 最新V1.0.0.94 (Discussion #133)
老哥你好，我也遇到了和你一样的问题，也是电量为0无法升级到v1.0.0.94。能否请教你一下具体做法？以及请问升级后电量显示会正常吗

**Comment by hhbbww83** [comment]
谢谢你的帮助！

yangyh75 ***@***.***> 于2024年10月3日周四 17:51写道：
